src/components/KeyboardSpacer/index.android.js

```diff
@@ -1,21 +0,0 @@
-/**
- * On Android the keyboard covers the input fields on the bottom of the view. This component moves the
- * view up with the keyboard allowing the user to see what they are typing.
- */
-import React from 'react';
-import {StatusBar} from 'react-native';
-import BaseKeyboardSpacer from './BaseKeyboardSpacer';
-import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
-
-const KeyboardSpacer = () => (
-    <BaseKeyboardSpacer
-        topSpacing={StatusBar.currentHeight}
-        keyboardShowMethod="keyboardDidShow"
-        keyboardHideMethod="keyboardDidHide"
-    />
-);
-
-KeyboardSpacer.propTypes = windowDimensionsPropTypes;
-KeyboardSpacer.displayName = 'KeyboardSpacer';
-
-export default withWindowDimensions(KeyboardSpacer);
```

Comment:
Why are we deleting this file? This fix is for mWeb chrome, and shouldn't affect behavior on Android native (AFAIK)

Reply:
understood! all good, thanks for linking everything and clarifying